language/move-prover/spec-lang/src/env.rs

```diff
@@ -70,7 +71,7 @@ pub const ABORTS_IF_IS_STRICT_PRAGMA: &str = "aborts_if_is_strict";
 pub const REQUIRES_IF_ABORTS: &str = "requires_if_aborts";
 
 /// Pragma indicating that the function will run smoke tests
-pub const SMOKE_TEST_PRAGMA: &str = "smoke_test";
+pub const ALWAYS_ABORTS_TEST_PRAGMA: &str = "always_aborts_test";
```

Comment:
Eventually, we should come up with a better name than "smoke test"

Reply:
This PR does this in a way, the pragma is now called "always_aborts_test". The code calls the thing now "test instrumentation".